sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -129,10 +129,13 @@ export default {
     };
 
     const addressList1 = (item: InstitutionLocationsDetails) => {
+      let addressString = item.data.address.provinceState
```

Comment:
We have a method to format the address, can we use it here?

Reply:
good catch 👍